specification/resources/resource-manager/Microsoft.Resources/stable/2016-06-01/subscriptions.json

```diff
@@ -312,6 +337,49 @@
         "nextLink"
       ],
       "description": "Tenant Ids information."
+    },
+    "Operation": {
+      "description": "Microsoft.Resources operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "properties": {
```

Comment:
Missing required "description":
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#exposing-available-operations

Reply:
You mean the display.description?

Reply:
Yes

Reply:
For display.description, I already had it. It is above display.properties